client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1712,12 +1714,12 @@
   "title.repack-details": "Repack Details",
   "title.sensor-details": "Sensor Details",
   "title.stock-line-details": "Stock Line Details",
-  "title.stock-on-hand": "Available Stock",
+  "title.stock-on-hand": "Stock on Hand",
```

Comment:
Since `our-soh` says `Our available Stock on Hand` should this also say available stock on hand? 🤔

Reply:
no, this is correct. it's only used by the item stats panel and that's now showing total soh and not available soh.